### PR TITLE
fix docstring in `get_acquisition_function`

### PR DESCRIPTION
**Pull request by jduerholt**
## Motivation

When working through the release notes for 0.9.0, I encountered this faulty docstring.
### Have you read the [Contributing Guidelines on pull requests](https://github.com/pytorch/botorch/blob/main/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

Not needed.

### CODE REVIEW

**Comment by jduerholt** [comment]
Btw, should this notebook also be updated at some point to the new constraint handling functionality for the qParEgo example?  https://github.com/pytorch/botorch/blob/main/tutorials/constrained_multi_objective_bo.ipynb

File: botorch/acquisition/utils.py [hunk 1]
```diff
@@ -65,13 +65,13 @@ def get_acquisition_function(
         constraints: A list of callables, each mapping a Tensor of dimension
             `sample_shape x batch-shape x q x m` to a Tensor of dimension
             `sample_shape x batch-shape x q`, where negative values imply
-            feasibility. Used only for qEHVI and qNEHVI.
+            feasibility. Used for all acquisition functions despite qSR and qUCB.
```
Comment (Balandat) [inline botorch/acquisition/utils.py:68]:
```suggestion
            feasibility. Used for all acquisition functions except qSR and qUCB.
```

File: botorch/acquisition/utils.py [hunk 2]
```diff
@@ -65,13 +65,13 @@ def get_acquisition_function(
...
         eta: The temperature parameter for the sigmoid function used for the
             differentiable approximation of the constraints. In case of a float the
             same eta is used for every constraint in constraints. In case of a
             tensor the length of the tensor must match the number of provided
             constraints. The i-th constraint is then estimated with the i-th
-            eta value. Used only for qEHVI and qNEHVI.
+            eta value. Used for all acquisition functions despite qSR and qUCB.
```
Comment (Balandat) [inline botorch/acquisition/utils.py:74]:
```suggestion
            eta value. Used for all acquisition functions except qSR and qUCB.
```

**Review by Balandat — COMMENTED** [review]
thanks!

**Comment by Balandat** [comment]
> Btw, should this notebook also be updated at some point to the new constraint handling functionality for the qParEgo example? https://github.com/pytorch/botorch/blob/main/tutorials/constrained_multi_objective_bo.ipynb

Yup good catch, it very much should.

**Pull request closed without merging**